src/applications/check-in/pre-check-in/pages/Introduction/IntroductionDisplay.jsx

```diff
@@ -112,9 +95,7 @@ const IntroductionDisplay = props => {
         onKeyDown={handleStart}
         onClick={handleStart}
       >
-        {isPreCheckInActionLinkTopPlacementEnabled
-          ? t('complete-pre-check-in')
-          : t('answer-questions')}
+        {t('answer-questions')}
```

Comment:
This should be complete-pre-check-in since this feature is on, and removing the toggle should just hard code it to that value.

Reply:
I'm wrong!